schemas/2018-08-20-preview/Microsoft.HealthcareApis.json

```diff
@@ -171,115 +137,100 @@
       "type": "object",
       "properties": {
         "origins": {
-          "readOnly": false,
           "oneOf": [
             {
               "type": "array",
-              "description": "The origins to be allowed via CORS.",
               "items": {
-                "anyOf": [
-                  {
-                    "type": "string",
-                    "description": "An origin for CORS.",
-                    "pattern": "^(?:(?:(?:[hH][tT][tT][pP](?:[sS]|))\\:\\/\\/(?:[a-zA-Z0-9-]+[.]?)+(?:\\:[0-9]{1,5})?|[*]))$"
```

Comment:
We will still need "pattern" field.

Reply:
Thanks for pointing this out. This seems like a bug in https://github.com/Azure/autorest.azureresourceschema when parsing string-valued array items. Will investigate further.